packages/graphql-transformers-e2e-tests/src/__tests__/KeyTransformer.e2e.test.ts

```diff
@@ -46,6 +46,7 @@ beforeAll(async () => {
     type Item @model
         @key(fields: ["orderId", "status", "createdAt"])
         @key(name: "ByStatus", fields: ["status", "createdAt"], queryField: "itemsByStatus")
+        @key(name: "ByCreatedAt", fields: ["createdAt", "status"], queryField: "itemsByCreatedAt")
```

Comment:
Can you add a test function below that queries this new index and verifies results?